templates/data/opensearch/support.html

```diff
@@ -0,0 +1,544 @@
+  track.
+{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1ovtXv8RBybw-flvttyqWOzZivUryQMP34T1UGe90HlY/edit
+{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-strip is-shallow u-no-padding--bottom">
+    <div class="p-section">
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <div class="u-hide--small u-vertically-center u-aspect-ratio--16-9"
+               style="margin-top: -55px">
+            {{ image(url="https://assets.ubuntu.com/v1/6deb3921-opensearch-logo.png",
+                        alt="Opensearch",
+                        width="198",
+                        height="186",
+                        hi_def=True,
+                        loading="lazy") | safe
+            }}
+          </div>
+        </div>
+        <div class="col-9 col-medium-4">
+          <div class="p-section--shallow">
+            <h1>OpenSearch support</h1>
+            <p>
+              We are experts in OpenSearch. From providing everyday support to offering hands-on assistance
+              for complex projects, our experienced database performance specialists are prepared to assist
+              you with any challenge.
+            </p>
+            <p>
+              Deal with issues faster and guarantee the ongoing performance of your deployments with 24/7/365
+              consultative OpenSearch support.
+            </p>
+          </div>
+          <hr />
+          <a class="p-button--positive"
+             href="/contact-us"
+             aria-controls="data-opensearch-modal">Contact us</a>
+          <a class="p-button"
+             href="https://assets.ubuntu.com/v1/6dc3440a-Charmed%20OpenSearch%20-%20Datasheet%20-%20amended%2004%2006%202024.pdf">Download
+          the datasheet<span class="u-off-screen">Download the Charmed OpenSearch datasheet</span></a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="is-fixed-width" />
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <h2 class="p-text--small-caps highlight">Why choose Canonical for OpenSearch support</h2>
+      </div>
+      <div class="col-9 col-medium-4">
+        <ul class="p-list--divided">
+          <li class="p-list__item u-no-padding">
+            <p class="p-heading--2 u-no-padding">Up to 10 years support and security maintenance</p>
+          </li>
+          <li class="p-list__item u-no-padding">
+            <p class="p-heading--2 u-no-padding">Low total cost of ownership</p>
+          </li>
+          <li class="p-list__item u-no-padding">
+            <p class="p-heading--2 u-no-padding">Experts in the full open source stack</p>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section u-fixed-width">
```

Comment:
No need to use u-fixed-width on a section. Please remove it and keep the class on hr. As an option you can move hr into the row div so that additional class won't be necessary